Bell's Inequality

```diff
@@ -0,0 +1 @@
+Markdown code describing the Physics of Bell's inequality
```

Comment:
add from already existing documentation

Reply:
On it. I was actually adding some content to this file, so shall I do that and then create a separate pull request as my second one for this?
And remove this blank file for time being?

Reply:
No this should be done in this PR.

Reply:
You mean adding from already existing documentation should be done in this PR or my own changes as well?

Reply:
Ideally both

Reply:
I have added the documentation from existing 3 files with my own into this.